aws/README.md

```diff
@@ -78,43 +64,20 @@ This is an example of the output you should see when you create the cluster:
-`eksctl utils associate-iam-oidc-provider --cluster openstudio-server --approve`
-
-2 ) Create your Amazon EBS CSI plugin IAM role with eksctl (replace "openstudio-server" if your cluster is named differently). 
-
-```bash 
-eksctl create iamserviceaccount \
-  --name ebs-csi-controller-sa \
-  --namespace kube-system \
-  --cluster openstudio-server \
-  --attach-policy-arn arn:aws:iam::aws:policy/service-role/AmazonEBSCSIDriverPolicy \
-  --approve \
-  --role-only \
-  --role-name AmazonEKS_EBS_CSI_DriverRole
-```
-3 ) Install the Add-On (replace \<YOUR ACCOUNT ID\> with your account id). An easy way to get your account id is to run the following cmd `eksctl get cluster openstudio-server -o yaml | grep arn`
-```bash 
-eksctl create addon --name aws-ebs-csi-driver --cluster openstudio-server --service-account-role-arn arn:aws:iam::<YOUR ACCOUNT ID>:role/AmazonEKS_EBS_CSI_DriverRole --force
+```bash
+eksctl create cluster -f eks_config_large.yaml
 ```
-
-4 ) Update kubernetes cluster __and__ add-on version in aws console.
-If you see update option in eks console for cluster and/or add-on of EBS. Just update both. Otherwise could cause issue like:
-
-The pvc could be keeping pending forever.  
-
-If you do `kubectl describe pvc <pvc-id>` it may raise issue:  
+Below is an example that will create an AWS EKS cluster for large workloads where the worker pods use spot instances. It still has 2 node groups, one for web related pods and the other for worker pods, but the instance types and the maximum number of nodes are larger. The web node group consists of 1 node of instance type `m7i.8xlarge` with max nodes = 2. The worker node group consists of 1 node of instance type `c7i.24xlarge` with max nodes = 50. This cluster is set to autoscale up to this max node amount. You can change the instance type and min and max node setting to your use case. More info on [AWS instance types](https://aws.amazon.com/ec2/instance-types/). Note that you will need to request an increase to your spot instance type quota at the account level. Go to [EC2 Quota Dashboard](console.aws.amazon.com/servicequotas/home/services/ec2/quotas) and search for "standard (A, C, D, H, I, M, R, T, Z) Spot". Then click on "All Standard (A, C, D, H, I, M, R, T, Z) Spot Instance Requests". Finally click on "Request increase at account level", enter 4000 for "Increase quota value", and click "Request". Note that this configuration did appear to have some issues, so it may need to be revised.
 
 ```bash
-caused by: AccessDenied: Not authorized to perform sts:AssumeRoleWithWebIdentity
-          status code: 403, request id: 60da95cf-c66e-471e-96a5-bd8915983baf
+eksctl create cluster -f eks_config_large-spot.yaml
 ```
 
+## EKS Add-On services
+
+As of Kubernetes version 1.23, to use EBS volumes you must install an EKS Add-On service called EBS CSI driver. However, this step is now handled with the eksctl cluster config files.
```

Comment:
can we provide a hyperlink to the relevant part of the config file?

Reply:
I added a hyperlink.